Marlin/ultralcd.cpp

```diff
@@ -2099,8 +2105,8 @@ void kill_screen(const char* lcd_msg) {
         #endif
       ;
       sprintf_P(UBL_LCD_GCODE, PSTR("G26 C B%i H%i P"), temp, custom_hotend_temp);
-      lcd_enqueue_command_sram("G28");
-      lcd_enqueue_command_sram(UBL_LCD_GCODE);
+      lcd_enqueue_commands_P(PSTR("G28"));
```

Comment:
If this is the only cause for `lcd_enqueue_commands_P` to exist, it might be better to simply allow this 4 byte string to reside in SRAM and move on.